lsl/data/lwana-ssmif.txt

```diff
@@ -996,7 +1340,136 @@ SNAP_DESI[2] 2
 # SNAP_INC[][] will be set to "UNK" ("unknown") for any boards/channels not identified
 # *** SNAP_INC[snap#][ch#] goes here:
 # Format is <M(iddle)|L(eft)>_<T(op)|B(ottom)>_<input>
-# FIXME JD: CT to provide this
+
+SNAP_INC[1][0]  M_B_3
```

Comment:
I think there is something a little off here.  `SNAP_INC[X][Y]` should have `X` in [1,2] and `Y` in [1,64] since they refer to the signal number of the SNAP.  The `(M|L)_(T|B)_Z` labels should have `Z` in [1,32] since they should correspond to the label on the dig. board.

Reply:
To clarify here on the 'Y' field. What I have currently is that the 'Y' field would have this pairwise inversion included in it where inputs [0..3] are cables ordered as [3,4,1,2]. Should this scheme also be imparted upon the [1,64] signals, then reset at input 65? The proposed reset in the example below would change [67,68,65,66] -> [3,4,1,2]

Ex. As is the Last 4 of board 1 and First 4 of board two would be this:
SNAP_INC[1][63]  L_T_28
SNAP_INC[1][64]  L_T_29
SNAP_INC[1][61]  L_T_30
SNAP_INC[1][62]  L_T_31
SNAP_INC[2][67]  L_B_0
SNAP_INC[2][68]  L_B_1
SNAP_INC[2][65]  L_B_2
SNAP_INC[2][66]  L_B_3